server/database/index.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+require('env2')('config.env');
+
+let dbURI = '';
+if (process.env.NODE_ENV === 'production') {
```

Comment:
you can use if else to add development env 
and create another database for development
it is not recommended to modify production database in development stage